### PR TITLE
Don't overwrite directories

### PR DESCRIPTION
**Pull request by Atlante45**

### CODE REVIEW

File: interface/resources/qml/AssetServer.qml [hunk 1]
```diff
@@ -74,6 +74,11 @@ Window {
     function doRenameFile(oldPath, newPath) {
         console.log("Renaming " + oldPath + " to " + newPath);
 
+        if (Assets.isKnownFolder(newPath)) {
+            box = errorMessageBox("Cannot override existing directory.");
```
Comment (birarda) [inline interface/resources/qml/AssetServer.qml]: this should be overwrite

File: interface/src/scripting/AssetMappingsScriptingInterface.cpp [hunk 2]
```diff
@@ -171,6 +171,19 @@ void AssetMappingsScriptingInterface::renameMapping(QString oldPath, QString new
     request->start();
 }
 
+bool AssetMappingModel::isKnownFolder(QString path) const {
```
Comment (birarda) [inline interface/src/scripting/AssetMappingsScriptingInterface.cpp:177]: shouldn't this be `isInsideKnownFolder`

Comment (Atlante45) [inline interface/src/scripting/AssetMappingsScriptingInterface.cpp:177]: No, we're checking if the new folder name already exist

Comment (birarda) [inline interface/src/scripting/AssetMappingsScriptingInterface.cpp:177]: Okay, right.

**Pull request merged**